knowledge/music/events/cochella_2023/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: music
+seed_examples:
+- answer: a music festival that took place over two weekends in mid-April 2023
+  question: What is coachella?
+- answer: The festival was headlined by Puerto Rican rapper Bad Bunny, South Korean girl group Blackpink, American singer Frank Ocean, and American rock band Blink-182.[a]
+  question: who headlined coachella_2023?
+- answer: Ocean's Coachella performance was harshly criticized from audience members, critics, and fans.[5] The Festive Owl, speaking to several sources, later stated that a group of people surrounding Ocean were intended to be ice skaters and had practiced for weeks
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!